docs/user-guide/faq.md

```diff
@@ -186,14 +188,19 @@ If you need to reference an old version of a dataset, you can find previous vers
 Data from AACR Project GENIE are provided in a [dedicated instance of cBioPortal](https://www.cbioportal.org/genie/). You can also download GENIE data from the [Synapse Platform](https://synapse.org/genie). Note that you will need to register before accessing the data. Additional information about AACR Project GENIE can be found on the [AACR website](https://www.aacr.org/Research/Research/Pages/aacr-project-genie.aspx).
 
 ### TCGA
-#### How does TCGA data in cBioPortal compare to TCGA data in Genome Data Commons?
-We do not currently load the mutation data from the GDC. Instead, we have the original mutation data generated by the individual TCGA sequencing centers. The source of the data is the Broad Firehose (or the publication pages for data that matches a specific manuscript). These data are usually a combination of two mutation callers, but they differ by center (typically a variant caller like MuTect plus an indel caller), and sequencing centers have modified their mutation calling pipelines over time.
+#### What are the TCGA studies sourced from the Genomic Data Commons (GDC)?
+The GDC TCGA studies mirror the [Cancer Gateway in the Cloud (ISB-CGC)](https://bq-search.isb-cgc.org/search?status=current) that is hosted on Google BigQuery, which in turn pulls data from GDC. Our [NCI-CRDC pipeline](https://github.com/cBioPortal/nci-crdc-pipeline) pulls data from ISB-CGC and transforms it into cBioPortal-formatted files. The resulting studies are intended to be a pure reflection of what is available inside ISB-CGC; we do not augment them with data from our legacy studies. For more information on how ISB-CGC handles GDC data, see [Programs and Data Sets](https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/Hosted-Data.html) and [GDC Overview](https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/GDC_top.html).
+
+#### How do the TCGA studies sourced from GDC compare to the TCGA PanCancer and Firehose Legacy studies?
```

Comment:
Right now we have this question and also "What are TCGA Firehose Legacy datasets and how do they compare to the publication-associated datasets and the PanCancer Atlas datasets?" below. Rather than 2 questions that each compare 3 of the different TCGA options, I think it would be clearer to combine into a single question which describes PanCan vs GDC vs publication vs Firehose Legacy. Can you integrate this content into the existing question below?